runtime/web_worker.rs

```diff
@@ -220,42 +247,24 @@ impl WebWorker {
       {
         let op_state = js_runtime.op_state();
         let mut op_state = op_state.borrow_mut();
-        op_state.put(RuntimeMetrics::default());
         op_state.put::<Permissions>(permissions);
         op_state.put(ops::UnstableChecker {
           unstable: options.unstable,
         });
       }
 
+      js_runtime.init_extension_ops().unwrap();
```

Comment:
Why unwrap?

Reply:
Could change it to an `.expect()`, but ultimately we should panic here if it fails, the worker constructors don't return `Result`, though they obviously could. But I would rather do a 2nd pass and flesh out/polish error handling in Extensions.